### PR TITLE
Draft of winter school report

### PR DESCRIPTION
**Pull request by karthik**
Could use a review from @danielskatz

### CODE REVIEW

File: content/blog/2020-01-29-winter-school-recap.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Our goal wasn't to repeat the same material covered in bootcamps and Software Carpentry style workshops, but to focus on the research software engineering skills that are not formally taught in any setting. These skills include best practices for packaging code as software, testing, collaborative software development, code review, and related topics such as licensing and archiving.  Two months before the workshop, we asked each student to select a software project related to their research to bring to the workshop. Our intent was to intermingle instructions with work time for students to apply these skills to their own projects.
```
Comment (danielskatz) [inline content/blog/2020-01-29-winter-school-recap.md]: "packaging code as software" seems awkward

File: content/blog/2020-01-29-winter-school-recap.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+### **Lessons learned**
+
+If anything it has become very clear to us that there is a strong demand for these skills, based on the demand for participation and general positive feedback from participants (more on that). We also found that, in this pilot, lectures and discussion ended up filling most of the time. We had originally scheduled roughly the same time for working on projects, but used much of that time for extended discussions. In the future we would like to run a longer workshop to allow students more time to work on their projects and apply the lessons they have learned in practice.
```
Comment (danielskatz) [inline content/blog/2020-01-29-winter-school-recap.md]: what does "(more on that)" mean?

File: content/blog/2020-01-29-winter-school-recap.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Despite our modest expectations for a first run of the course, the response from the participants was overwhelmingly positive. Here’s what a few participants had to say about the workshop:
+
+ \
+–”I really can't say enough good things about this super empowering workshop! You did an amazing job identifying the things I didn't know that I didn't know, and teaching them at a level that was immediately actionable in my work.”
+
+ – “Thank you so much to everyone for taking this amazing initiative to teach young scientists on software sustainability.”
+
+ – “Thank you for putting together this winter-school, it was super useful to me and I'm looking forward to applying everything I learned to my future projects and to go deeper into the topics that were covered.”
+
+We look forward to teaching a longer version of this school once the full institute is conceived.
```
Comment (danielskatz) [inline content/blog/2020-01-29-winter-school-recap.md]: maybe end with a different work - conceived seems too similar to conceptualized - initiated? funded? starts?

File: content/blog/2020-01-29-winter-school-recap.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+date: 2020-01-29
+author: "Karthik Ram, Kyle Niemeyer, and Jeffrey Carver"
+---
+
+![Winter School](https://imgur.com/njacNOL.png)
+
+
+<br><br>
+
+In late December 2019, we ran our first ever URSSI school on research software engineering. We began accepting applications in  July and received an overwhelming response to our call for applications. For the 30 participant slots available, we received 169 applications, meaning we had a challenging time selecting the participants and had to turn away a large number of interested researchers. While the selection committee used multiple criteria to evaluate and select participants, successful applicants already had some experience with Python programming, Git, and Unix skills, which was necessary to benefit from the workshop.
```
Comment (danielskatz) [inline content/blog/2020-01-29-winter-school-recap.md]: what were the selection criteria?

**Comment by danielskatz** [comment]
some minor comments added.
Additionally, you might:
- provide a link to the workshop
- provide a link to the agenda
- provide a link to an attendee list
- provide some stats about the attendees, eg. experience, career status, gender, diversity, geography
- say something similar about the instructors and helpers (and say that the authors were the instructors and name the helpers)
- add something about how the material was taught, eg. using carpentry-style lessons?
- say that the course material is available (and if so, where)

**Comment by karthik** [comment]
We didn't ask to make the participant list public so I wont do that after the fact.
I did add links to agenda, materials, and some stats about career stage, institutions, and fields (we did not ask about gender so we should not infer and report on that).

**Comment by karthik** [comment]
@kyleniemeyer Would you like to take a look and add anything? 
Current preview is here: http://urssi.us/blog/2020/01/29/report-from-the-urssi-winter-school-pilot/

File: content/blog/2020-01-29-winter-school-recap.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+We also found that it was really important to have supplemental instructors (“helpers”), beyond the three lead organizers, who could help teach some material and/or answer specific questions. Through some advertising and direct requests, we found six teaching assistants from diverse backgrounds who brought a wealth of technical and domain knowledge.
+
+
+### **Feedback**
+
+Despite our modest expectations for a first run of the course, the response from the participants was overwhelmingly positive. Here’s what a few participants had to say about the workshop:
+
+–”I really can't say enough good things about this super empowering workshop! You did an amazing job identifying the things I didn't know that I didn't know, and teaching them at a level that was immediately actionable in my work.”
```
Comment (kyleniemeyer) [inline content/blog/2020-01-29-winter-school-recap.md]: minor formatting fix: `–”I` to ` – “I`

**Comment by kyleniemeyer** [comment]
@karthik @danielskatz we could name the helpers and give their background/institution, if that provides any value, since that information was already public. I can suggest some text on that if desired. 

Otherwise, looks good.

**Comment by danielskatz** [comment]
This seems useful to me

File: content/blog/2020-01-29-winter-school-recap.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+date: 2020-01-29
+author: "Karthik Ram, Kyle Niemeyer, and Jeffrey Carver"
+---
+
+![Winter School](https://imgur.com/njacNOL.png)
+
+
+<br><br>
+
+In late December 2019, we ran our first ever [URSSI school on research software engineering](https://github.com/si2-urssi/winterschool#tentative-schedule). We began accepting applications in July and received an overwhelming response to our call for applications. For the 30 participant slots available, we received 169 applications, meaning we had a challenging time selecting the participants and had to turn away a large number of interested researchers. Our students came from 22 different universities/research institutes, mostly graduate students (19 students, 8 postdocs, and 1 professional researcher), and from fields such as astrophysics, bioinformatics, chemical engineering and neuroscience. While the selection committee used multiple criteria to evaluate and select participants, successful applicants already had some experience with Python programming, Git, and Unix skills, which was necessary to benefit from the workshop.
```
Comment (danielskatz) [inline content/blog/2020-01-29-winter-school-recap.md]: maybe remove the anchor from the URL for this link?

File: content/blog/2020-01-29-winter-school-recap.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+Our goal wasn't to repeat the same material covered in bootcamps and Software Carpentry style workshops, but to focus on the research software engineering skills that are not formally taught in any setting. These skills include best practices for converting a loose collection of scripts as installable software, testing, collaborative software development, code review, and related topics such as licensing and archiving.  Two months before the workshop, we asked each student to select a software project related to their research to bring to the workshop. Our intent was to intermingle instructions with work time for students to apply these skills to their own projects. All of our materials are available on [URSSI's GitHub](https://github.com/si2-urssi/winterschool#tentative-schedule).
```
Comment (danielskatz) [inline content/blog/2020-01-29-winter-school-recap.md]: for what it's worth, some of these topics are taught in other places, such as the HEP training offered by the HEP software foundation, MolSSI, and Katy previously developed a testing module for software carpentry, though it's not hosted by them.  Maybe change "not formally taught in any setting" to "generally not formally taught in most settings"

**Comment by danielskatz** [comment]
by "this", I meant @kyleniemeyer's suggestion of adding names/backgrounds for helpers

**Comment by karthik** [comment]
@kyleniemeyer Please do!

> we could name the helpers and give their background/institution, if that provides any value, since that information was already public. I can suggest some text on that if desired.

**Comment by karthik** [comment]
I pushed more fixes. Once @kyleniemeyer writes something about the helpers, we can post this.

**Comment by karthik** [comment]
@kyleniemeyer I added the names of the helpers. Is there anything else you want to add before this is posted?

**Comment by karthik** [comment]
Draft of post: https://deploy-preview-36--urssi.netlify.com/blog/2020/01/29/report-from-the-urssi-winter-school-pilot/

**Comment by danielskatz** [comment]
This looks good to me.  @kyleniemeyer might be on a plane now - let's wait until he responds if possible.

**Comment by karthik** [comment]
He just checked in over chat. Will wait.

**Comment by kyleniemeyer** [comment]
I landed! Ship it!

**Pull request merged**